php_swoole_grpc_bench/server.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Grpc\Parser;
+use Helloworld\HelloReply;
+use Helloworld\HelloRequest;
+
+require __DIR__ . '/vendor/autoload.php';
+
+$http = new Swoole\Http\Server('0.0.0.0', 50051, SWOOLE_BASE);
+$http->set([
+    'trace_flags' => 0,
+    'log_file' => '/dev/null',
+    'log_level' => 5,
+    'open_http2_protocol' => true
+]);
+
+$http->on('request', function ($request, $response) {
+
+    try {
+        $request_message = new HelloRequest;
+        $request_message->mergeFromString(Parser::unpack($request->rawContent()));
+
+        $response_message = new HelloReply();
+        $response_message->setMessage("Hello {$request_message->getName()}");
```

Comment:
You don't need to concatenate with "Hello", in other benchmarks we just set the same message as received. Perhaps it should've been called "EchoRequest" but it's kind of too late for that. :)

Reply:
Ok, I fixed that :)

Reply:
Thank you!